src/server/auth/server/testing/auth_test.go

```diff
@@ -2298,3 +2298,15 @@ func TestGetRobotTokenErrorNonAdminUser(t *testing.T) {
 	require.YesError(t, err)
 	require.Matches(t, "needs permissions \\[CLUSTER_AUTH_GET_ROBOT_TOKEN\\] on CLUSTER", err.Error())
 }
+
+func TestCreateProject(t *testing.T) {
```

Comment:
We should also test the case where a user doesn't have PROJECT_CREATE permission